homeassistant/components/esphome/strings.json

```
@@ -69,6 +78,10 @@
     "api_password_deprecated": {
       "title": "API Password deprecated on {name}",
       "description": "The API password for ESPHome is deprecated and the use of an API encryption key is recommended instead.\n\nRemove the API password and add an encryption key to your ESPHome device to resolve this issue."
+    },
+    "service_calls_not_allowed": {
+      "title": "{name} is not permitted to call Home Assistant services",
+      "description": "The ESPHome device attempted to make a Home Assistant service call, but this functionality is not enabled.\n\nIf you trust this device and want to allow access for it to make Home Assistant service calls, you can enable this functionality in the options flow."
```

Small simplification. 

```suggestion
      "description": "The ESPHome device attempted to make a Home Assistant service call, but this functionality is not enabled.\n\nIf you trust this device and want to allow it to make Home Assistant service calls, you can enable this functionality in the options flow."
```